src/scripts/check.sh

```diff
@@ -21,7 +22,7 @@ Check_For_ShellCheck() {
     if ! command -v shellcheck &> /dev/null
     then
         echo "Shellcheck not installed"
-        exit 1
```

Comment:
Don't we want this? Maybe you accidentally removed it?

Reply:
Ha, yes. Accidentally reverted that fixing the test failures. Should be correct now!